README.md

```diff
@@ -1,7 +1,59 @@
-# DevOps-Course-Starter-Exercise-2
-Second Exercise with the whole lot refactored.
+# DevOps Apprenticeship: Project Exercise
```

Comment:
You seem to have a .README.md.swp file that needs to be deleted, from an aborted edit of the README